setup.py

```diff
@@ -49,6 +49,8 @@ def read(fname):
         "idna>=2.8",
         "psycopg2>=2.8.4",
         "importlib_metadata",
+        "pywebpush>=1.13.0",
+        "py-vapid>=1.7.0",
```

Comment:
As someone who has no added libraries before, do we care about the licenses of these or anything of that nature? (Probably a question for @matrix-org/synapse-core)

Reply:
[py-vapid](https://pypi.org/project/py-vapid/) and [pywebpush](https://pypi.org/project/pywebpush/) are both MPL2. Should transitive dependencies be checked?